EpiAware/src/EpiObsModels/LatentDelay.jl

```diff
@@ -64,13 +64,16 @@ Generates observations based on the `LatentDelay` observation model.
 
 "
 @model function EpiAwareBase.generate_observations(obs_model::LatentDelay, y_t, Y_t)
-    @assert length(obs_model.pmf)<=length(Y_t) "The delay PMF must be shorter than or equal to the observation vector"
+    first_Y_t = findfirst(!ismissing, Y_t)
```

Comment:
I'm abit confused that this is looking for the possibility of `missing` values in `Y_t`? What is the use case for this? Different length latent models?

Reply:
OK I can see you mention this in the new docstrings.

Reply:
This is so you can stack `LatentDelay` (i.e it preserves composability). Maybe another solution here but not sure what it is